client/src/components/dashboard/AddColumn.js

```diff
@@ -0,0 +1,36 @@
+import React, {Fragment, useState} from 'react';
+import {IconButton} from '@material-ui/core/';
+import AddCircleOutlineIcon from '@material-ui/icons/AddCircle';
+
+import AddColumnDialogForm from '../dashboardForms/AddColumnDialogForm';
+
+const AddColumn = (props) => {
+  const {boardId, setLoadBoard} = props;
+  const [open, setOpen] = useState(false);
+
+  const handleClickOpen = (event) => {
+    event.preventDefault();
+    setOpen(true);
+  };
+  return (
+    // eslint-disable-next-line react/jsx-fragments
+    <Fragment>
```

Comment:
Please try not to disable eslint rules. They help enforce a cleaner style more often than not. In this particular case, [this eslint rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-fragments.md) wants you to use [React's shorthand syntax for `React.Fragment `components](https://reactjs.org/docs/fragments.html#short-syntax).

Reply:
I dont remember exactly what I went through. It was the last moment fix. When I was trying quick fix, Fragment component changed to fragments (not React.Fragment). And the fragments appeared on my browser console.  Thats why I disabled it. I will consider your point next time. Thanks for letting me know